pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -0,0 +1,188 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package nodelabeller
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Node-labeller config", func() {
+	BeforeSuite(func() {
+		os.Mkdir("/tmp/cpu_map", 0777)
+	})
+
+	AfterSuite(func() {
+		os.RemoveAll("/tmp/cpu_map")
+	})
+
+	It("should return correct cpu file path", func() {
+		cpuName := "Penryn"
+		path := getPathCPUFefatures(cpuName)
+		Expect(path).To(Equal("/var/lib/kubevirt-node-labeller/cpu_map/x86_Penryn.xml"), "cpu file path is not the same")
```

Comment:
It would be better to hold all directories/file paths as constants (In both code and tests).
Maybe as a pre-formatted string and then use it like this: 
https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/rest/subresource_test.go#L52

https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/rest/subresource_test.go#L156